incl/minishell.h

```diff
@@ -40,6 +40,8 @@
 # define EXIT_EXECVE 		5
 # define EXIT_EXECNOTFOUND	127
 
+extern int	g_signal;
+
```

Comment:
Global variable to store the signal number. Has to be named "g_[name]". 

Norminette now detects it and prints a note (not error!) that it is in a particular file and that its use should be justifiable.